src/libstd/io/mod.rs

```diff
@@ -1761,6 +1761,35 @@ impl<T> Take<T> {
     #[stable(feature = "rust1", since = "1.0.0")]
     pub fn limit(&self) -> u64 { self.limit }
 
+    /// Sets the number of bytes that can be read before this instance will
+    /// return EOF. This is the same as constructing a new `Take` instance, so
+    /// the amount of bytes read or the previous limit value do not affect the
```

Comment:
"do not affect the effect" is a somewhat confusing phrasing. Also missing period.